integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -172,7 +275,12 @@ def clean_content(self) -> str:
                         "Updated link url from %s to %s", href, translated_url
                     )
 
-        # Scan for media files in content and replace alt texts
+    def process_images(self, content: HtmlElement) -> None:
```

Comment:
I could find a better title other than this generic one. Do you have any suggestions for a better one?

Reply:
Since this function currently only does one thing, something specific like `fix_alt_texts` could work